cumulusci/tasks/metadata_etl/tests/test_help_text.py

```diff
@@ -0,0 +1,405 @@
+        <externalId>false</externalId>
+        <label>FERPA Approved</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Bluth__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Indicates that the Contact is allowed to receive information protected by HIPPA and other privacy laws, regulations, and policies.</description>
+        <externalId>false</externalId>
+        <label>HIPPA Approved</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+</CustomObject>
+"""
+
+STANDARD_OBJECT_XML = b"""<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>Buster</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Indicates that the Contact is allowed to receive information protected by FERPA and other privacy laws, regulations, and policies.</description>
+        <externalId>false</externalId>
+        <label>FERPA Approved</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Bluth</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Indicates that the Contact is allowed to receive information protected by HIPPA and other privacy laws, regulations, and policies.</description>
+        <externalId>false</externalId>
+        <label>HIPPA Approved</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+</CustomObject>
+"""
+
+
+class TestAddPicklistValues:
+    def test_add_single_object_help_text(self):
+        task = create_task(
+            AddHelpText,
+            {
+                "api_version": "47.0",
+                "entries": [
+                    {"object_field": "MyObject.Buster__c", "help_text": "buster"}
+                ],
+            },
+        )
+
+        # Validate that the first sObject has one picklist changed
+        tree = metadata_tree.fromstring(OBJECT_XML)
+        result = task._transform_entity(tree, "MyObject")
+        test_elem = result.find("fields", fullName="Buster__c")
+
+        assert test_elem is not None
+        assert test_elem.inlineHelpText.text == "buster"
+        # Validate that the sObject alters only the custom field listed
+        test_elem = result.find("fields", fullName="Bluth__c")
+        assert test_elem is not None
+        assert (
+            test_elem.inlineHelpText.text
+            == "Indicates that the Contact is allowed to receive information protected by HIPPA and other privacy laws, regulations, and policies."
```

Comment:
You might want to trim down the text values in your unit tests to help make them more readable/accessible.